ArduCopter/mode.h

```diff
@@ -274,6 +277,7 @@ class Mode {
     GCS_Copter &gcs();
     void set_throttle_takeoff(void);
     uint16_t get_pilot_speed_dn(void);
+    bool allows_weathervaning_auto(void) const;
```

Comment:
let's move this somewhere else in the .h file because I don't think it's one of this "pass-through functions"..

Reply:
done